lib/php/lib/Protocol/TProtocol.php

```diff
@@ -181,7 +181,7 @@ abstract public function readString(&$str);
      * The skip function is a utility to parse over unrecognized date without
      * causing corruption.
      *
-     * @param TType $type What type is it
+     * @param int $type What type is it
```

Comment:
@param int $type What type is it (defined in TType::class)

Reply:
@sveneld I'm not sure what your comment implies? Also I don't understand the patch.

Reply:
Before PHP7 phpdoc was used for defining types in php. This docblock is just for code understanding. Based on this doc block IDE give a suggestions for argument. TType is not a possible type for argument which will come to this method, integer value will come. That's why @rtm-ctrlz offer to write a real type of this function. And I suggest to add comment where the possible values is defined.

When I will return to my plan of migrating thrift to php7 with type declaration i will update this code.